lib/plugins/aws/deploy/compile/events/stream/README.md

```diff
@@ -0,0 +1,56 @@
+# Compile Stream Events
```

Comment:
I don't think we need this README. Its basically just a copy of the documentation so the whole file can be removed.